django/db/models/deletion.py

```diff
@@ -117,6 +122,26 @@ def add_field_update(self, field, value, objs):
             model, {}).setdefault(
             (field, value), set()).update(objs)
 
+    def add_restricted_objects(self, field, objs):
+        if not objs:
+            return
+        model = objs[0].__class__
+        self.restricted_objects.setdefault(
```

Comment:
Line line is preferred here (lines up to 119 chars are allowed if it helps readability.